tutorial/config/task/random_continuous__id_assoc_ks.yaml

```diff
@@ -0,0 +1,41 @@
+
+defaults:
+  - identify_associations_ks_schema
+
+batch_size: 10
+
+num_refits: 10 
+sig_threshold: 0.05
+
+target_dataset: random.continuous.proteomics 
+target_value: plus_std
+save_refits: True
+
+model:
+  categorical_weights: ${weights:${data.categorical_inputs}}
+  continuous_weights: ${weights:${data.continuous_inputs}}
```

Comment:
I suggest to move these interpolations (`${etc}`) to the `identify_associations_ks_schema` definition, because they are not meant to be overridden and here they are likely going to confuse users.

Reply:
ok

Reply:
the file currently does not exist, I guess @mpielies ?

Reply:
So we don't know how to add this to the schema here: https://github.com/RasmussenLab/MOVE/blob/2cb9aa55d8a05a3b3e0476bb918fb60056ea5de6/src/move/conf/schema.py#L189-L207

Do you know?

Reply:
Two options:
A) Directly edit the schema: For example, add `categorical_weights: list[int] = "${weights:${data.categorical_inputs}}" # type: ignore` to that class definition.
B) Create a new default config: This goes in `src/move/conf/task`. These configs serve as defaults and then user-made configs should name them as defaults.

I have made a commit using approach B.